TLM/TLM/TrafficManagerMod.cs

```diff
@@ -94,6 +94,12 @@ public class TrafficManagerMod : IUserMod {
             InGameHotReload = InGame();
 
             HarmonyHelper.EnsureHarmonyInstalled();
+
+#if DEBUG
+            const bool installHarmonyASAP = false; // set true for fast testing
+            if (installHarmonyASAP)
+                HarmonyHelper.DoOnHarmonyReady(delegate () { Patcher.Create().Install(); });
```

Comment:
Can you use lambdas instead?
this delegate () syntax is out of vogue since c# 3.0.